bundles/org.eclipse.equinox.common/src/org/eclipse/core/runtime/Path.java

```diff
@@ -90,10 +90,13 @@ public final class Path implements IPath, Cloneable {
 	 * segment and device delimiters for the local file system are
 	 * also respected.
 	 *
-	 * @param pathString the portable string path
+	 * @param pathString the operating-system specific string path
+	 * @return the IPath representing the given OS specific string path
 	 * @see IPath#toPortableString()
 	 * @since 3.1
+	 * @deprecated use {@link IPath#fromOSString(String)} instead
 	 */
+	@Deprecated(since = "3.18", forRemoval = true)
```

Comment:
I'm not happy about `forRemoval = true` here and below.
I see ~250 references in my SDK workspace and ~50 references in our product.

While the move from Path to IPath might be OK, but deleting this here seem to be not really justified, considering amount of existing code that uses the old API.

Reply:
Yes, I dread dealing with such things.  All my projects want to work on older versions of Eclipse and all my projects want to be warning free tomorrow, so I will need to create utilities to isolate such things if there really is a threat of removal.  I am not likely to be the only one with such a concern and need.

Reply:
I agree with others.  We should not mark this for removal at this time.